src/app/Main/Body/InfoPage/InfoPage.model.tsx

```diff
@@ -29,26 +29,59 @@ export type AmrKey =
 
 export const microorganismNames: Record<string, JSX.Element> = {
     Salm: <i>Salmonella</i>,
-    Campy: <i>Campylobacter</i>,
+    SalmSpp: (
```

Comment:
I recommend to move these names to a separate constants file.

Reply:
Done.